chart/values.yaml

```diff
@@ -13,6 +13,7 @@ images: # images and tag to pull for each component of the stack
 controller:
   deleteOnly: false # enable delete-only mode
   metricsSink: noop # metrics driver (noop or datadog)
+  notifierSinks: noop # notifier sinks list (slack, noop, both) - ex: "slack,noop"
```

Comment:
Following my comment on the configmap file, you should be able to customize those values from there.